1-org/envs/shared/org_policy.tf

```diff
@@ -18,89 +18,31 @@ locals {
+    "constraints/compute.vmExternalIpAccess"            = "list",
+    "constraints/compute.skipDefaultNetworkCreation"    = "boolean",
+    "constraints/compute.restrictXpnProjectLienRemoval" = "boolean",
+    "constraints/compute.requireOsLogin"                = "boolean",
+  }
 }
 
-
 /******************************************
   Compute org policies
 *******************************************/
 
-module "org_disable_nested_virtualization" {
-  source          = "terraform-google-modules/org-policy/google"
-  version         = "~> 3.0"
-  organization_id = local.organization_id
-  folder_id       = local.folder_id
-  policy_for      = local.policy_for
-  policy_type     = "boolean"
-  enforce         = "true"
-  constraint      = "constraints/compute.disableNestedVirtualization"
-}
-
-module "org_disable_serial_port_access" {
-  source          = "terraform-google-modules/org-policy/google"
-  version         = "~> 3.0"
-  organization_id = local.organization_id
-  folder_id       = local.folder_id
-  policy_for      = local.policy_for
-  policy_type     = "boolean"
-  enforce         = "true"
-  constraint      = "constraints/compute.disableSerialPortAccess"
-}
-
-module "org_compute_disable_guest_attributes_access" {
-  source          = "terraform-google-modules/org-policy/google"
-  version         = "~> 3.0"
-  organization_id = local.organization_id
-  folder_id       = local.folder_id
-  policy_for      = local.policy_for
-  policy_type     = "boolean"
-  enforce         = "true"
-  constraint      = "constraints/compute.disableGuestAttributesAccess"
-}
-
-module "org_vm_external_ip_access" {
+module "org_compute_policies" {
+  for_each        = local.policies
   source          = "terraform-google-modules/org-policy/google"
   version         = "~> 3.0"
   organization_id = local.organization_id
   folder_id       = local.folder_id
   policy_for      = local.policy_for
-  policy_type     = "list"
-  enforce         = "true"
-  constraint      = "constraints/compute.vmExternalIpAccess"
-}
-
-module "org_skip_default_network" {
-  source          = "terraform-google-modules/org-policy/google"
-  version         = "~> 3.0"
-  organization_id = local.organization_id
-  folder_id       = local.folder_id
-  policy_for      = local.policy_for
-  policy_type     = "boolean"
-  enforce         = "true"
-  constraint      = "constraints/compute.skipDefaultNetworkCreation"
-}
-
-module "org_shared_vpc_lien_removal" {
-  source          = "terraform-google-modules/org-policy/google"
-  version         = "~> 3.0"
-  organization_id = local.organization_id
-  folder_id       = local.folder_id
-  policy_for      = local.policy_for
-  policy_type     = "boolean"
-  enforce         = "true"
-  constraint      = "constraints/compute.restrictXpnProjectLienRemoval"
-}
-
-module "org_shared_require_os_login" {
-  source          = "terraform-google-modules/org-policy/google"
-  count           = var.enable_os_login_policy ? 1 : 0
```

Comment:
Thanks for the pull request.

This condition ,`count           = var.enable_os_login_policy ? 1 : 0`, was added because there are some limitations on using OS Login as listed in the 1-org README.md and reproduced bellow:

**Note:** It is possible to enable an organization policy for [OS Login](https://cloud.google.com/compute/docs/oslogin/manage-oslogin-in-an-org) with this module.
OS Login has some [limitations](https://cloud.google.com/compute/docs/instances/managing-instance-access#limitations).
If those limitations do not apply to your workload/environment you can choose to enable the OS Login policy by setting variable `enable_os_login_policy` to `true`.

To be able to maintain this behavior it would be necessary to conditionally add  the OS Login constraint `"constraints/compute.requireOsLogin"` to the map, something like

```
locals {
...
  policies = {
    "constraints/compute.disableNestedVirtualization"   = "boolean",
    "constraints/compute.disableSerialPortAccess"       = "boolean",
    "constraints/compute.disableGuestAttributesAccess"  = "boolean",
    "constraints/compute.vmExternalIpAccess"            = "list",
    "constraints/compute.skipDefaultNetworkCreation"    = "boolean",
    "constraints/compute.restrictXpnProjectLienRemoval" = "boolean",

  }
  os_login_policy = {
    "constraints/compute.requireOsLogin" = "boolean",
  }
  resolved_policies = merge(local.policies, var.enable_os_login_policy ? local.os_login_policy : {})
}

```

Reply:
@daniel-cit I was thinking we could just comment it inline with issue and just add a section in the readme instructing users to tweak this part.
```hcl
 policies = {
    "constraints/compute.disableNestedVirtualization"   = "boolean",
    ...
   #  "constraints/compute.requireOsLogin" = "boolean" // disabled due to https://cloud.google.com/compute/docs/instances/managing-instance-access#limitations
    
    }
```
This way, we can also inform them of other policies like https://github.com/terraform-google-modules/terraform-example-foundation/pull/271 without introducing a flag per policy.

Reply:
It would also work, it is just important that in the README we list which are the optional ones. 

It will also be necessary to remove the commented org policies from the list in the integration test:

https://github.com/terraform-google-modules/terraform-example-foundation/blob/f87ed1661bec7c2d2c75ea091d1cf5db6ac6943a/test/integration/org/controls/gcloud_policy.rb#L18